CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ provide as much detail and context as possible.
 
 ## Pull requests
 
-Good pull requests—patches, improvements, new features—are a fantastic
+Good pull requests - patches, improvements, and new features - are a fantastic
```

Comment:
It comes down to style/preference. As a rule, em dashes do indeed not *require* spaces. However, most newspapers that employ the Associated Press styleguide do include spaces. Personally, I find em dashes more readable with spaces either side.